lib/linter.js

```diff
@@ -31,23 +31,18 @@ function buildErrorMessage(filePath, error) {
   return message;
 }
 
-class Linter {
+export default class Linter {
   constructor(_options) {
     let options = _options || {};
 
     this.options = options;
     this.console = options.console || console;
     this.workingDir = options.workingDir || process.cwd();
-
-    this.loadConfig();
```

Comment:
Config loading can't happen in `Linter` class constructor anymore since it's async. It now happens when calling `Litner` class public functions like `verify`.